nemo/collections/common/data/lhotse/cutset.py

```diff
@@ -47,6 +53,193 @@ def read_cutset_from_config(config) -> Tuple[CutSet, bool]:
     return cuts, is_tarred
 
 
+KNOWN_DATASET_CONFIG_TYPES = frozenset(("nemo", "nemo_tarred", "lhotse", "lhotse_shar", "txt", "txt_pair", "group"))
```

Comment:
Why "lhotse_shar" why not "lhotse_tarrred" ?

Reply:
It's the name of the format https://colab.research.google.com/github/lhotse-speech/lhotse/blob/master/examples/04-lhotse-shar.ipynb